src/Adapter/DbSelect.php

```diff
@@ -122,7 +122,9 @@ public function count()
         $result    = $statement->execute();
         $row       = $result->current();
 
-        $this->rowCount = (int) $row[self::ROW_COUNT_COLUMN_NAME];
+        $this->rowCount = isset($row[self::ROW_COUNT_COLUMN_NAME])
+            ? (int) $row[self::ROW_COUNT_COLUMN_NAME]
+            : (int) $row['c'];
```

Comment:
Should the column name not be based on the constant?

Reply:
updated.